### PR TITLE
Update javaDoc and ug

### PR DESCRIPTION
**Pull request by wgzesg**

### CODE REVIEW

File: src/main/java/workout/workoutmanager/WorkoutManagerParser.java [hunk 1]
```diff
@@ -81,6 +89,13 @@
         return test;
     }
 
+    /**
+     * Parse the given input into a integer index.
```
Comment (tienkhoa16) [inline src/main/java/workout/workoutmanager/WorkoutManagerParser.java]: Parses?

Comment (tienkhoa16) [reply src/main/java/workout/workoutmanager/WorkoutManagerParser.java]: an integer index?

File: src/main/java/workout/workoutmanager/WorkoutManagerParser.java [hunk 2]
```diff
@@ -91,13 +106,18 @@ public static int parseIndex(String args) throws NotANumberException {
         return index;
     }
 
+    /**
+     * Parses user input into a list of predicates which will be used to limit period of record being listed.
+     * @param args user input
```
Comment (tienkhoa16) [inline src/main/java/workout/workoutmanager/WorkoutManagerParser.java]: can consider adding a blank line between description and @param to follow [coding standards](https://se-education.org/guides/conventions/java/basic.html#comments)? And add punctuation behind each parameter description also. It's in the "note in particular" session.

**Review by tienkhoa16 — COMMENTED** [review]
Great work in enhancing the documentation

File: src/main/java/workout/workoutmanager/WorkoutManagerParser.java (hunk 1 above)
Comment (wgzesg) [reply src/main/java/workout/workoutmanager/WorkoutManagerParser.java]: ohh tkss. Will fix that

File: src/main/java/workout/workoutmanager/WorkoutManagerParser.java (hunk 2 above)
Comment (wgzesg) [reply src/main/java/workout/workoutmanager/WorkoutManagerParser.java]: Icic. Will check on that

File: docs/UserGuide.md [hunk 3]
```diff
@@ -568,11 +610,24 @@ __Format:__ `clear`
 This command is dangerous as you will not be able to 
 recover the data.
 After typing this command, you will be asked to reconfirm it by typing in
-`YES`. Else the action will be aborted. 
+`YES`. Any other input will abort the clearing.
```
Comment (tienkhoa16) [inline docs/UserGuide.md:613]: inputs?

**Pull request merged**